ui/src/app/utils/leo-runtime-initializer.tsx

```diff
@@ -134,9 +138,22 @@ export class LeoRuntimeInitializer {
   private resumeCount = 0;
   private serverErrorCount = 0;
   private initializeStartTime?: number;
+  private runtime?: Runtime;
   // The latest runtime retrieved from getRuntime. If the last getRuntime call returned a NOT_FOUND
   // response, this will be null.
-  private currentRuntime?: Runtime;
+  private _currentRuntime?: Runtime;
```

Comment:
opt: This leading "_" is not a naming convention I've seen anywhere else in our codebase, though I see why you're using it here.

Relatedly, I think the property setter has too many side effects. I might prefer a normal method.

Reply:
What does "opt" mean in your commenting?

Reply:
Sorry, my review short-hand (pasting this in from a previous explanation):

**nit**: Minor comment on style/spelling. I use this to communicate that this comment is not critical to approval and should generally be dealt with at a lower priority than other comments. If resolving the comment results in tons of unanticipated work or other complications, it can probably be abandoned.

**opt**: Optional. You can use your judgement to accept/reject this comment (of course, this is true for any PR comment, but here I'm conveying my own expectations). It is either an alternative approach you could consider, or it is based on personal subjective opinion - as opposed to agreed upon style guide / best practices.

**FYI**: no action required, just making a comment